scripts/get-docker-compose.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -eu
+which 'docker-compose' > /dev/null 2>&1
+NO_HYPHEN=$?
```

Comment:
what is the idea of this?

Reply:
There are two names for "docker compose". One is "docker compose" with a space, as a docker subcommand, and the other one, "docker-compose" with a dash. I personally can't run "docker compose", I need the dash "docker-compose", but I know some of our colleagues can't use the dash. I guess I could do an alias, but in my opinion this is more solid and portable. The dash will solve my problem, whereas this will solve everyone's problem.